pkgs/development/libraries/md4c/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "md4c";
+  version = "release-0.4.6";
+  src = fetchFromGitHub {
+    owner = "mity";
+    repo = "md4c";
+    rev = version;
```

tag prefixes go here
```suggestion
    rev = "release-${version}";
```